cpp/models/ode_secir/parameters_io.cpp

```diff
@@ -74,12 +74,12 @@ IOResult<void> read_confirmed_cases_data(
     });
     if (max_date_entry == rki_data.end()) {
         log_error("RKI data file is empty.");
```

Comment:
If you want to, you could think about testing these uncovered lines but its ok i guess. There actually is an empty file in the test data directory "test_empty_file.json".

Reply:
I would leave that out for now, as it doesn't really affect the Issue. 
But if I improve the coverage again in the future, I'll keep it in mind.
Also, thanks for your effort in the review :)